src/algorithms.jl

```diff
@@ -818,8 +593,7 @@ for Alg in [:Exp4, :EPIRK4s3A, :EPIRK4s3B, :EPIRK5s3, :EXPRB53s3, :EPIRK5P1, :EP
             iop)
     end
 end
-struct SplitEuler <:
```

Comment:
why was this removed?

Reply:
It should end up with the split methods group, with CNAB2 and such. What's the dependency?

Reply:
![image](https://github.com/user-attachments/assets/5596a7e1-b3d3-4017-b73d-aae08db16a6c)

when I went through low_order_rk_caches.jl file I saw the splitEuler Cache here so I thought that splitEuler might be an algorithm of low_order_rk. That was the reason

Reply:
That was a good guess but ultimately not the right one. There should be a SplitMethods which has CNAB2 and such, and SplitEuler is kind of a smaller test method that goes with that set.